src/sensor_motion.py

```diff
@@ -0,0 +1,28 @@
+import RPi.GPIO as GPIO
+from time import sleep
+
+GPIO.setmode(GPIO.BCM)
+GPIO.setwarnings(False)
+pir = 16
+GPIO.setup(pir, GPIO.IN)
+
+print('Initializing Sensor...')
+sleep(2)
+print("Let's do this")
+
+def getMotion():
+    if GPIO.input(pir):
+        # print("MOTION DETECTED")
+        return True
+    
+    elif not GPIO.input(pir):
+        # print("<no motion>")
+        return False
+
+    elif KeyboardInterrupt: 
+        GPIO.cleanup() #reset GPIO
+        print("Program ended")
+
+# while True:
+#     getMotion()
+#     sleep(.2)
```

Comment:
Remove this file, because this PR is related to reading the temperature and humidity from the sensor.